app/serializers/v2/profile_serializer.rb

```diff
@@ -7,7 +7,7 @@ class V2::ProfileSerializer
 
   attributes :assessment_answers
 
-  belongs_to :person, serializer: ::V2::PersonSerializer
+  belongs_to :person, serializer: ::V2::PeopleSerializer
```

Comment:
I can see the intent here to avoid introducing a potentially breaking change to the existing API with the new category relationship, but it's also stretching things a little to use the `PeopleSerializer` to return a single person. Perhaps this needs a little renaming to better align... or potentially could we check with suppliers that they can tolerate the introduction of a new relationship without breaking things?

So either rename something like this

* existing `person_serializer.rb` <- retain the same name
* new `person_serializer.rb` <- rename to `person_with_category_serializer.rb`

or... just stick with a single person serializer for V2 that includes the new `category` relationship. If this is an idiomatic belongs_to or has_one model relationship (potentially with lazy loading if needed) then I wouldn't think that introduces any performance penalty as we only perform the explicit Nomis lookup when invoking the new `GET /people/:id` endpoint.

profile_serializer.

Reply:
Discussed offline -

Reply:
the serializers have been renamed 👍